src/server/req_holdjob.c

```diff
@@ -302,7 +302,7 @@ req_releasejob(struct batch_request *preq)
 
 	old_hold = get_jattr_long(pjob, JOB_ATR_hold);
 	rc = job_attr_def[(int)JOB_ATR_hold].
-		at_set(&pjob->ji_wattr[(int)JOB_ATR_hold],
+		at_set(get_jattr(pjob, JOB_ATR_hold),
```

Comment:
There shouldn't be a single at_set() call that's outside of setters. Convert this to set_jattr_generic()